File: app/src/main/java/me/srikavin/quiz/model/UserProfile.java
package me.srikavin.quiz.model;

import android.media.Image;

import com.google.gson.annotations.Expose;

public class UserProfile {
    @Expose
    private String id;
    @Expose
    private Image avatar;
    @Expose
    private String username;

    public Image getAvatar() {
        return avatar;
    }

    public String getUsername() {
        return username;
    }
}

File: app/src/main/java/me/srikavin/quiz/model/UserProfile.kt
package me.srikavin.quiz.model

import android.media.Image

import com.google.gson.annotations.Expose

class UserProfile {
    @Expose
    private val id: String? = null
    @Expose
    val avatar: Image? = null
    @Expose
    val username: String? = null
}
